src/ast/ddl.rs

```diff
@@ -640,6 +640,68 @@ impl fmt::Display for AlterIndexOperation {
     }
 }
 
+/// An `ALTER TYPE` (`Statement::AlterType`) operation
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum AlterTypeOperation {
+    Rename {
+        new_name: Ident,
+    },
+    AddValue {
+        if_not_exists: bool,
+        value: Ident,
+        position: AlterTypeOperationAddValuePosition,
+    },
+    RenameValue {
+        from: Ident,
+        to: Ident,
+    },
+}
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum AlterTypeOperationAddValuePosition {
```

Comment:
```suggestion
pub enum AlterTypeAddValuePosition {
```
maybe an opportunity to shorten the name, since AddValue is already an operation